calculated_policies/aws_ec2_approved_image_tag/variables.tf

```diff
@@ -0,0 +1,17 @@
+variable "smart_folder_title" {
```

Comment:
Variables now have a file called output.tf that displays their values. 

These values are then sent back to the end user at the end of the apply for their reference.

Reply:
To clarify, variables are defined here.

The values of the variables are defined in `output.tf`